extensions/discovery-and-network/horizontal-pod-autoscalers.yaml

```diff
@@ -146,9 +187,73 @@ data:
             - Pod
             - Deployment
             - ReplicaSet
+            - StatefulSet
+            - ReplicationController
+        - simple: true
+          required: true
+          path: name
+          widget: Resource
+          resource:
+            scope: namespace
+            kind: Pod
+            version: v1
+          visibility: spec.scaleTargetRef.kind = "Pod"
+          overwrite: false
+        - simple: true
+          var: separator
+          value: ""
+        - simple: true
+          required: true
+          path: name
+          widget: Resource
+          resource:
+            scope: namespace
+            kind: Deployment
+            group: apps
+            version: v1
+          visibility: spec.scaleTargetRef.kind = "Deployment"
+          overwrite: false
+        - simple: true
+          var: separator
+          value: ""
+        - simple: true
+          required: true
+          path: name
+          widget: Resource
+          resource:
+            scope: namespace
+            kind: ReplicaSet
+            group: apps
+            version: v1
+          visibility: spec.scaleTargetRef.kind = "ReplicaSet"
+          overwrite: false
+        - simple: true
+          var: separator
+          value: ""
         - simple: true
           required: true
           path: name
+          widget: Resource
+          resource:
+            scope: namespace
+            kind: StatefulSet
+            group: apps
+            version: v1
+          visibility: spec.scaleTargetRef.kind = "StatefulSet"
+          overwrite: false
+        - simple: true
+          var: separator
+          value: ""
+        - simple: true
+          required: true
+          path: name
+          widget: Resource
+          resource:
+            scope: namespace
+            kind: ReplicationController
+            version: v1
+          visibility: spec.scaleTargetRef.kind = "ReplicationController"
+          overwrite: false
```

Comment:
We don't have ReplicationController in Busola, so this could be normal Text, and probably we need additional normal Text field if someone will provide some different resource